src/test/java/org/jclouds/vcloud/director/v1_5/internal/BaseVCloudDirectorApiLiveTest.java

```diff
@@ -207,23 +205,32 @@ public User randomTestUser(String prefix, Reference role) {
                .role(role).password("password").build();
    }
 
+   @Override
+   public Properties setupProperties() {
+      Properties overrides = super.setupProperties();
+      
+      vAppTemplateUrn = this.setIfTestSystemPropertyPresent(overrides, this.provider + ".vapptemplate-id");
```

Comment:
why you moved only some `*Urn` up into `setupProperties` from `initTestParametersFromPropertiesOrLazyDiscover`